code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -990,12 +990,45 @@
 			H.reagents.remove_reagent(chem.type, chem.volume - 5)
 			to_chat(H, "<span class='warning'>The excess milk is dripping off your bones!</span>")
 		H.heal_bodypart_damage(1.5,0, 0)
-		H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM)
+		H.reagents.remove_reagent(chem.type, chem.metabolization_rate)
 		return TRUE
-
 	if(chem.type == /datum/reagent/toxin/bonehurtingjuice)
+		H.adjustStaminaLoss(7.5, 0)
 		H.adjustBruteLoss(0.5, 0)
-		H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM)
+		if(prob(20))
+			switch(rand(1, 3))
+				if(1)
+					var/list/possible_says = list("oof.", "ouch!", "my bones.", "oof ouch.", "oof ouch my bones.")
+					H.say(pick(possible_says), forced = /datum/reagent/toxin/bonehurtingjuice)

```

Comment:
`H.say("[pick(each individual item here, not prebuilt in list)]", ....)`

Reply:
doesn't really matter, does it?

Reply:
saves a var, pretty minor